examples/Web/Views/_ViewStart.cshtml

```diff
@@ -0,0 +1,3 @@
+@{
+    Layout = "_Layout";

```

Comment:
_aaah

Reply:
this is by convention, i inherited it from autogenerated code.